### PR TITLE
Add Hatsune Miku and Future Stars - Project Mirai

### PR DESCRIPTION
**Pull request by maplecomb**
Adds Hatsune Miku and Future Stars - Project Mirai.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/hatsune-miku-and-future-stars-project-mirai/game.dat [hunk 1]
```diff
@@ -0,0 +1,9 @@
+title = "Hatsune Miku and Future Stars - Project Mirai"
```
Comment (CodingKoopa) [inline games/hatsune-miku-and-future-stars-project-mirai/game.dat]: The " - " should be changed to a ": ".

File: games/hatsune-miku-and-future-stars-project-mirai/game.dat [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+description = "A rhythm game created by Sega and Crypton Future Media."
```
Comment (CodingKoopa) [inline games/hatsune-miku-and-future-stars-project-mirai/game.dat]: >A rhythm game

This should be rephrased as "Hatsune Miku and Future Stars: Project Mirai is a..."

Comment (CodingKoopa) [inline games/hatsune-miku-and-future-stars-project-mirai/game.dat]: There should be another sentence here about the gameplay. Remember to write it in objective terms, e.g. "in the game the player..."  or "the game consists of..."

File: games/hatsune-miku-and-future-stars-project-mirai/game.dat [hunk 3]
```diff
@@ -0,0 +1,9 @@
+title = "Hatsune Miku and Future Stars: Project Mirai"
+description = "Hatsune Miku and Future Stars: Project Mirai is a rhythm game created by Sega and Crypton Future Media. This game featured the Chance Circle System. Buttons will appear around the border of a circle, and a pointer will appear from the center of the circle extending all the way outside the circle's border. It will then turn in a clockwise or counter-clockwise manner according to the song and when the pointer passes the button, the player has to press the face button on the 3DS."
```
Comment (CodingKoopa) [inline games/hatsune-miku-and-future-stars-project-mirai/game.dat]: Nit: "This game featured..." should be "This game features..."

**Comment by maplecomb** [comment]
I have added a wiki entry in this repo.

**Comment by robzombie91** [comment]
isnt this just a copy of https://citra-emu.org/game/hatsune-miku-project-mirai-dx/

**Pull request merged**